vowpalwabbit/automl.cc

```diff
@@ -597,27 +597,32 @@ VW::LEARNER::base_learner* automl_setup(VW::setup_base_i& stack_builder)
   int priority_challengers;
   bool keep_configs = false;
 
-  option_group_definition new_options("Debug: automl reduction");
+  option_group_definition new_options("Automl");
   new_options
       .add(make_option("automl", max_live_configs)
                .necessary()
                .keep()
                .default_value(3)
-               .help("set number of live configs"))
+               .help("Set number of live configs"))
       .add(make_option("global_lease", global_lease)
                .keep()
                .default_value(10)
-               .help("set initial lease for automl interactions"))
-      .add(make_option("cm_type", cm_type).keep().default_value("interaction").help("set type of config manager"))
+               .help("Set initial lease for automl interactions"))
+      .add(make_option("cm_type", cm_type)
+               .keep()
+               .default_value("interaction")
+               .one_of({"interaction"})
+               .help("Set type of config manager"))
       .add(make_option("priority_type", priority_type)
                .keep()
                .default_value("none")
-               .help("set function to determine next config {none, least_exclusion}"))
+               .one_of({"none", "least_exclusion"})
```

Comment:
we should be able to delete the if at 631 in this file

Reply:
We still need the `if` at 631 to set the function pointer based on the input, but we can delete the `THROW` after the `else if`